### DISCUSSION TITLE
DAI 2023-2024 - My Docker Compose application - Walid Slimani

### DISCUSSION THREAD

**Discussion by SlWa99** [heig-vd-dai-course/heig-vd-dai-course#125] [Show and tell]
My repo : https://github.com/SlWa99/my-custom-dockerfile
My package : https://github.com/SlWa99/my-custom-dockerfile/pkgs/container/my-custom-dockerfile

![image](https://github.com/heig-vd-dai-course/heig-vd-dai-course/assets/94690298/0de432d8-2780-4b96-8e18-66d7fc7275db)

**Comment by ludelafo** [comment]
It seems the project is private as I cannot access it. Could you add us to the members of your repository so we could check your work?

**Reply by SlWa99** [reply]
Normally, you can access it. It is now public

**Reply by ludelafo** [reply]
Thank you!

Everything looks good to me, however, I would add a gitignore files to ignore the config and data volumes to avoid any personal file leaks.
